packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -5,53 +5,37 @@
 %global repo_orgname theforeman
 %global repo_name foreman_ygg_worker
 %global yggdrasil_libexecdir %{_libexecdir}/yggdrasil
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-%global yggdrasil_worker_conf_dir %{_root_sysconfdir}/yggdrasil/workers
+%global yggdrasil_worker_conf_dir %{_sysconfdir}/yggdrasil/workers
 
 %global goipath         github.com/%{repo_orgname}/%{repo_name}
 
-%if 0%{?rhel} > 7 && ! 0%{?fedora}
-%define gobuild(o:) \
-        GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback libtrust_openssl ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -compressdwarf=false -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags'" -a -v %{?**};
-%else
-%if ! 0%{?gobuild:1}
-%if 0%{?suse_version}
-%define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now '" -a -v %{?**};
-%else
-%define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '" -a -v %{?**};
-%endif
-%endif
-%endif
-
 Name: foreman_ygg_worker
 Version: 0.3.0
 Summary: Worker service for yggdrasil that can act as pull client for Foreman Remote Execution
-Release: 1%{?dist}
+Release: 2%{?dist}
 License: MIT
 
 Source0: https://github.com/%{repo_orgname}/%{repo_name}/releases/download/v%{version}/%{repo_name}-%{version}.tar.gz
 Url: https://github.com/%{repo_orgname}/%{repo_name}/
 
-# EL7 doesn't define go_arches
-%if ! 0%{?go_arches:1}
-%define go_arches %{ix86} x86_64 %{arm} aarch64 ppc64le
-%endif
 ExclusiveArch: %{go_arches}
```

Comment:
Opensuse does not define `go_arches` from what I can tell, they define `go_exclusivearches`:

```
%go_exclusivearch \
ExclusiveArch: aarch64 %ix86 x86_64 %arm ppc64 ppc64le s390x riscv64
```
I would try the following:

```
%if 0%{?suse_version}
%go_exclusivearch
%else
ExclusiveArch: %{go_arches}
%endif

Reply:
That's not too bad. Overall I think the spec is already a lot cleaner

Reply:
> I would try the following:

I think I tried that, but let's give that one more try

Reply:
Hrm, doesn't look like it
```
Start: buildsrpm
Start: rpmbuild -bs
error: line 23: Unknown tag: %{go_exclusivearch}
Building target platforms: x86_64
Building for target x86_64
Finish: rpmbuild -bs
```
https://download.copr.fedorainfracloud.org/results/@theforeman/client-nightly-staging-scratch-7d545e01-9b9c-5fae-a7e1-414cc8b054c9/opensuse-leap-15.5-x86_64/08549528-foreman_ygg_worker/builder-live.log.gz